doc/api/async_hooks.md

```diff
@@ -712,6 +712,75 @@ never be called.
 * Returns: {number} The same `triggerAsyncId` that is passed to the
 `AsyncResource` constructor.
 
+## Class: AsyncContext
+
+This class is used to create asynchronous state between callbacks and promise chain. It allows you to keep variables throughout the lifetime of a web request or any other asynchronous duration. It is similar to what ThreadLocal storage is in other languages.
```

Comment:
We usually wrap at 80 characters per line :) Also, typo: `promise chains`
```

Reply:
Good catch. Just fixed it!